website/content/blog/2024-12-01-a-year-in-review.md

```diff
@@ -0,0 +1,100 @@
+---
+title: A Year in Review
+date: "2024-12-01"
+excerpt: Details about the educational release of Old School Adventure
+tags: foundation, source, educational
+author:  MonoGame Foundation
+image: /images/blog/cover/monogame_foundation.png
+ogImage: /images/blog/cover/monogame_foundation.png
+---
+
+# What a year it's been!
+
+It has been a year since we announced that we received a very generous donation from Re-Logic and the formation of the MonoGame Foundation. A lot has happened.
+
+### So far in the last year we:
+
+* Now have recurring Patreon subscriptions of $788/month  
```

Comment:
I would say...  "almost $2000 a month".

This includes patreons as well as GitHub sponsors and Re-logics monthly $1000.